src/components/Select/components/Listbox/Listbox.scss

```diff
@@ -0,0 +1,154 @@
+@import '../../../../../styles/mixins';
+@import '../../../variables';
+
+$block: '.#{$ns}select-listbox';
+$popupBlock: '.#{$ns}popup';
+$s-height: '28px';
+$s-hor-padding: '8px';
+$m-height: '28px';
+$m-hor-padding: '8px';
+$l-height: '32px';
+$l-hor-padding: '12px';
+$xl-height: '36px';
+$xl-hor-padding: '12px';
+
+#{$block} {
+    --yc-popup-border-width: 0;
+
+    overflow: hidden;
+    box-shadow: 0px 6px 20px rgba(0, 0, 0, 0.15);
+
+    &__container {
+        display: flex;
+        max-height: 90vh;
+        margin: 4px 0;
+        overflow-y: auto;
+
+        // this selector used to increase specificity
+        #{$popupBlock}#{$block} & {
+            border-radius: 0;
+        }
+    }
+
+    &__group-label {
+        box-sizing: border-box;
+        position: relative;
+        width: 100%;
+        font-weight: 500;
+        white-space: nowrap;
+        text-overflow: ellipsis;
```

Comment:
We cannot use `@include overflow-ellipsis()` cause of we do not need style `overflow: hidden`

Reply:
It works without it?

Reply:
I added two styles instead
```css
white-space: nowrap;
text-overflow: ellipsis;
```

Reply:
Hmm.. no. I`ll fix that